### ISSUE TITLE
User Guide: Clarify which executable accepts command-line switches

### ISSUE THREAD

**Issue by elliott94** [nvaccess/nvda#11621]
### Is your feature request related to a problem? Please describe.

Section 16.2 of the User Guide details the command-line switches that are accepted by NVDA. Whilst the majority of these need to be added as part of the command on the main "nvda.exe" executable, several of these ("--install", "--install-silent" and "--enable-start-on-logon=True|False") need to be run as part of the argument when launching the main NVDA download package. In addition (to me, at least) it's not clear whether the "--create-portable" and "--create-portable-silent" switches need to be run on the main download package or the main NVDA executable itself. It'd be useful if this could be clarified to prevent any confusion.

**Comment by XLTechie** [comment]
Perhaps a separate section for installer commandline options?

**Comment by lukaszgo1** [comment]
I agree this may not be obvious but every command line option can be specified both when running the launcher and when running main NVDA executable. Launcher simply passes its command line parameters to the nvda.exe therefore nothing stops you from passing "--install " to the nvda.exe
